src/test/ui/function-pointer/function-pointer-comparison-issue-54685.rs

```diff
@@ -0,0 +1,29 @@
+// run-pass
```

Comment:
- The issue happened on the release build, we should tweak the optimization level here.
- The patch is available since LLVM 12, which means we need an additional flag.

```suggestion
// min-llvm-version: 12.0
// compile-flags: -C opt-level=3
// run-pass
```

Reply:
Thanks. That was very helpful.

Reply:
Applied the change.